internal/loggers/template_store.go

```diff
@@ -18,13 +19,17 @@ const (
 	TemplateURLsFile   = "/opt/autopi/template-urls.json"
 	DeviceSettingsFile = "/opt/autopi/device-settings.json"
 	VehicleInfoFile    = "/opt/autopi/vehicle-info.json"
+	DBCFile            = "/opt/autopi/dbc-settings.dbc"
 )
 
 //go:generate mockgen -source template_store.go -destination mocks/template_store_mock.go
 type TemplateStore interface {
 	ReadVINConfig() (*models.VINLoggerSettings, error)
 	WriteVINConfig(settings models.VINLoggerSettings) error
 
+	ReadDBCFile() (*string, error)
```

Comment:
I see that we are only reading the dbc file but not writing. How are we going to provision it, or are we supposed to do it now?

Reply:
looks like i missed that

Reply:
now it should save locally too